src/components/action-bar/action-bar.types.ts

```diff
@@ -0,0 +1,14 @@
+import { MenuItem } from '../menu/menu.types';
+
+export interface ActionBarItem extends MenuItem {
+    /**
+     * A non-changing value to uniquely identify each item.
+     */
+    id?: string;
```

Comment:
Why is this needed? We already have `value` from `MenuItem` that's more flexible and easier to use for storing `action.command` in our case.

Reply:
I'm not sure if the ID is needed. But we have an ID for a similar case in `limel-dock` 🤷‍♂️ 
I thought this was used to identify which action was clicked.
However, the menuItem types don't have any `id`s. We're sending the same ids to the menu too. So they have no effect there. I don't know how all of this is connected.

Reply:
> Will it always be unique? For bulk actions for instance we will use the same command ID several times right?

No, command id will not always be unique. Another good reason for removing id here I think. It would be annoying if you think you have to generate a unique id when you don't have one.

What you should do is just provide any useful unique value as `actionBarItem.value`.

Reply:
> I thought this was used to identify which action was clicked.

Yes, we did do that with dock, but `DockItem` doesn't have `value` (as it doesn't extend `MenuItem`).

Reply:
OK, then I'll remove that from types and all examples

Reply:
⚡ done